src/Analyzers/Core/Analyzers/NewLines/ConsecutiveStatementPlacement/AbstractConsecutiveStatementPlacementDiagnosticAnalyzer.cs

```diff
@@ -80,7 +80,10 @@ private void ProcessBlockLikeStatement(SyntaxTreeAnalysisContext context, Report
 
             // Grab whatever comes after the close brace.  If it's not the start of a statement, ignore it.
             var nextToken = endToken.GetNextToken();
-            var nextTokenContainingStatement = nextToken.Parent!.FirstAncestorOrSelf<TExecutableStatementSyntax>();
+            if (nextToken.Parent is null)
+                return;
+
+            var nextTokenContainingStatement = nextToken.Parent.FirstAncestorOrSelf<TExecutableStatementSyntax>();
```

Comment:
This is simpler IMO since there is a null check already for `nextTokenContainingStatement`. But you can simply ignore if you like the way it's currently written.

```suggestion
            var nextTokenContainingStatement = nextToken.Parent?.FirstAncestorOrSelf<TExecutableStatementSyntax>();
```

Reply:
works for me.  thanks!